### ISSUE TITLE
FloatBar Error

### ISSUE THREAD

**Issue by Metallicow** [wxWidgets/Phoenix#976]
**Operating system**:
Win7 Ult x64
wxPython-demo-4.0.4a1.dev3929+55c51b25
Python 2.7.15

**Description of the problem**:
Traceback error. Open demo. Navigate to floatbar demo. Start demo. Click on tool buttons.

```
C:\wxPython-demo-4.0.4a1.dev3929+55c51b25\demo>demo.py
('SetTitle', 'Floating!')
Floating!
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\wx-4.0.4-msw-phoenix\wx\lib\floatbar.py",
line 259, in OnMouse
    self.ReleaseMouse()
wx._core.wxAssertionError: C++ assertion "!wxMouseCapture::stack.empty()" failed
 at ..\..\src\common\wincmn.cpp(3319) in wxWindowBase::ReleaseMouse(): Releasing
 mouse capture but capture stack empty?
(1L, True, 1, True)
('SetTitle', '')
```

**Comment by RobinD42** [comment]
We shoudl probably just remove that module. It hasn't been touched for 15 years or so (other than whitespace changes and such) and probably hasn't worked in nearly that long.

**Comment by Metallicow** [comment]
I wouldn't say just delete it, But I myself find no use for it personally. Is there any certain way you keep statistics? Maybe move it to different repo maybe so that it is a history widget or some such.(Ex: old_stuff repo) After all the last one floated around for 82 years. http://www.sj-r.com/news/20180829/boom-old-meredosia-bridge-imploded Just keep in mind I wont be around at that time it goes down.

**Comment by RobinD42** [comment]
Closed by #1021
